service/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
     // Cromwell dependencies
     implementation('org.broadinstitute:cromwell_2.13:85') {
         exclude group: 'bio.terra', module: 'workspace-manager-client'
+        exclude group: 'mysql', module: 'mysql-connector-java'
```

Comment:
The GPL-2 license was flagged as a potential issue here, but if we're using this for the Cromwell client (rather than deploying a full Cromwell server in Axon) it seems like we would never be talking to a mysql db, and this dependency is safe to exclude. Is that right?

Reply:
Axon doesn't need to speak to mysql, cromwell will.